setup.py

```diff
@@ -28,7 +28,7 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-sdk>=1.35.0",
-        "pennylane>=0.25.1,<=0.27.0",
+        "pennylane==0.28.0",

```

Comment:
Note that PL version `0.29.0` was released this week :+1:

Reply:
I shall be working on a follow up here to support this. There are some cases where an upgrade is needed.

Reply:
As a follow up, the latest PL version is `0.29.1` due to an issue with `autoray`.

Reply:
Updated to reflect this and the changes with `>=0.29.0`